lib/rbac/access.rb

```diff
@@ -26,16 +26,17 @@ def process
         full_acls.each do |item|
           Rails.logger.debug("Found ACL: #{item}")
           @acls << item if regexp.match(item.permission)
+          @approver_acls << item if approver_regexp.match(item.permission)
+        end
 
-          unless @admin
+        unless @admin
```

Comment:
I don't understand the changes in this file. The old code looks fine to me.

Reply:
Old one loops on all acls, i.e `full_acls`. The right way should only loop on `acls`.